### DISCUSSION TITLE
where do i put the compiled .dlls

### DISCUSSION THREAD

**Discussion by valina354** [nillerusr/source-engine#316] [Q&A]
im making a mod based on this and cant figure out

**Comment by stephen-cusi** [comment]
`./waf install --destdir=./mymod`

**Reply by GreMirid** [reply]
I created mod by manually modifing .py scripts and wsscript files & creating need vpc files. It's not so hard (more of this just coping existing vpcs and pasting need data info inside).
![image](https://github.com/nillerusr/source-engine/assets/56092546/f4082ffd-5685-4005-8671-7cb6bd9b6d26)
![image](https://github.com/nillerusr/source-engine/assets/56092546/c5e0925e-7aed-4480-8ff8-0515b1125696)
![image](https://github.com/nillerusr/source-engine/assets/56092546/ccb93ddb-5120-4a5b-87c5-42e17b6a6365)

AND after that executed waf script in console:
`
.\waf.bat --64bits configure -T release --prefix=build/out/ --enable-opus --build-games=project07
`
